pkg/plugin/driver/daemonset/daemonset_test.go

```diff
@@ -608,60 +616,89 @@ func TestExpectedResults(t *testing.T) {
 				{NodeName: "node3", ResultType: "myPlugin"},
 				{NodeName: "node4", ResultType: "myPlugin"},
 			},
-			p: pluginWithAffinity(nil),
+			p: pluginWithNodeFilters(nil, nil),
 		}, {
-			desc: "Filters for label exists",
+			desc: "Affinity: Filters for label exists",
 			expect: []plugin.ExpectedResult{
 				{NodeName: "node2", ResultType: "myPlugin"},
 				{NodeName: "node3", ResultType: "myPlugin"},
 				{NodeName: "node4", ResultType: "myPlugin"},
 			},
-			p: pluginWithAffinity([]corev1.NodeSelectorRequirement{
+			p: pluginWithNodeFilters(nil, []corev1.NodeSelectorRequirement{
 				{Key: "foo", Operator: corev1.NodeSelectorOpExists},
 			}),
 		}, {
-			desc: "Filters for label does not exist",
+			desc: "Affinity: Filters for label does not exist",
 			expect: []plugin.ExpectedResult{
 				{NodeName: "node1", ResultType: "myPlugin"},
 			},
-			p: pluginWithAffinity([]corev1.NodeSelectorRequirement{
+			p: pluginWithNodeFilters(nil, []corev1.NodeSelectorRequirement{
 				{Key: "foo", Operator: corev1.NodeSelectorOpDoesNotExist},
 			}),
 		}, {
-			desc: "Filters for label value in",
+			desc: "Affinity: Filters for label value in",
 			expect: []plugin.ExpectedResult{
 				{NodeName: "node2", ResultType: "myPlugin"},
 				{NodeName: "node3", ResultType: "myPlugin"},
 			},
-			p: pluginWithAffinity([]corev1.NodeSelectorRequirement{
+			p: pluginWithNodeFilters(nil, []corev1.NodeSelectorRequirement{
 				{Key: "foo", Operator: corev1.NodeSelectorOpIn, Values: []string{"bar", "baz"}},
 			}),
 		}, {
-			desc: "Filters for label value not in",
+			desc: "Affinity: Filters for label value not in",
 			expect: []plugin.ExpectedResult{
 				{NodeName: "node1", ResultType: "myPlugin"},
 				{NodeName: "node4", ResultType: "myPlugin"},
 			},
-			p: pluginWithAffinity([]corev1.NodeSelectorRequirement{
+			p: pluginWithNodeFilters(nil, []corev1.NodeSelectorRequirement{
 				{Key: "foo", Operator: corev1.NodeSelectorOpNotIn, Values: []string{"bar", "baz"}},
 			}),
 		}, {
-			desc: "Can combine filters as union",
+			desc: "Affinity: Can combine selctor terms as union",
 			expect: []plugin.ExpectedResult{
 				{NodeName: "node1", ResultType: "myPlugin"},
 				{NodeName: "node2", ResultType: "myPlugin"},
 				{NodeName: "node4", ResultType: "myPlugin"},
 			},
-			p: pluginWithAffinity([]corev1.NodeSelectorRequirement{
-				{Key: "foo", Operator: corev1.NodeSelectorOpNotIn, Values: []string{"bar", "baz"}},
-				{Key: "foo", Operator: corev1.NodeSelectorOpIn, Values: []string{"bar"}},
```

Comment:
This test case was inconsistent with the behavior of Kubernetes scheduler (fixed)